node-runner-cli/config/KeyDetails.py

```diff
@@ -5,19 +5,9 @@
 class KeyDetails(BaseConfig):
     keyfile_path: str = Helpers.get_default_node_config_dir()
     keyfile_name: str = "node-keystore.ks"
```

Comment:
We might want to read this from env vars as users might not use the defualt vaule

Reply:
the values for this can get changed using command line arguments or modifying the config. I believe a third way is not necessary.